engine/docker.go

```diff
@@ -39,5 +40,6 @@ func newDockerClient(server *autoscaler.Server) (docker.APIClient, error) {
 		},
 	}
 	host := fmt.Sprintf("https://%s:2376", server.Address)
-	return docker.NewClient(host, api.DefaultVersion, client, nil)
+	new_client, err := docker.NewClient(host, api.DefaultVersion, client, nil)
```

Comment:
Go style guide prefers camel case, so you could call this something like `dockerClient`, to differentiate from the http client defined above.